### DISCUSSION TITLE
Add SSH Key to server with Key Name from Vito in Comment

### DISCUSSION THREAD

**Discussion by athphane** [vitodeploy/vito#319] [Ideas]
We already put a name for the SSH key when we add to Vito. How about when adding the key to the server we include a comment on top of the key containing the Key name from Vito.?

Something like this

```
# athphane ssh key 
ssh-ed25519 my-super-secret-key key-comment
```

Assuming "athphane ssh key" is the name given in Vito.

I think this would be helpful when you're in the server and looking at the `authorized_keys` file for whatever reason. Actually something that I was doing today.

**Comment by joglomedia** [comment]
@athphane the SSH public key added to Vito is generated from your local machine, right? You can add a key comment using the `-C` option with ssh-keygen.
